capstone-rs/src/capstone.rs

```diff
@@ -365,6 +365,44 @@ impl Capstone {
         }
     }
 
+    /// Get the registers are which are read to and written to, in that order.
+    pub fn regs_access(&self, insn: &Insn) -> Option<CsResult<(Vec<RegId>, Vec<RegId>)>> {
+        if cfg!(feature = "full") {
+            Some(Ok(unsafe {
+                let mut regs_read_count: u8 = 0;
+                let mut regs_write_count: u8 = 0;
+
+                let mut regs_write = [0u16; 64];
```

Comment:
please pull the 64 constant to a constant with a name like `MAX_NUM_REGISTERS`. You can add a comment that the Python bindings use the same value.